src/js/components/List/stories/RenderedList.js

```diff
@@ -32,6 +32,7 @@ export const RenderedList = () => (
           {item.location}
         </Text>
       )}
+      itemKey="key"
```

Comment:
```suggestion
      itemKey="entry"
```
Can we change this to map to the entry property in data?